apis/medusa/v1alpha1/medusa_types.go

```diff
@@ -143,4 +143,10 @@ type MedusaClusterTemplate struct {
 
 	// Provides all storage backend related properties for backups.
 	StorageProperties Storage `json:"storageProperties,omitempty"`
+
+	// Certificates for Medusa if client encryption is enabled in Cassandra.
+	// The secret must be in the same namespace as Cassandra and must contain three keys: "rootca.crt", "client.crt_signed" and "client.key".
+	// See https://docs.datastax.com/en/developer/python-driver/3.20/security/ for more information on the required files.
```

Comment:
3.25 is the latest version fyi. Or are you sticking to the exact version used by Medusa?

Reply:
not specifically. I'll put `latest` instead of a specific version, that should do it.